tests/unittest_brain_numpy_core_einsumfunc.py

```diff
@@ -0,0 +1,72 @@
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/astroid/blob/main/CONTRIBUTORS.txt
+
+from __future__ import annotations
+
+import pytest
+
+try:
+    import numpy  # pylint: disable=unused-import
+
+    HAS_NUMPY = True
+except ImportError:
+    HAS_NUMPY = False
+
+from astroid import builder
+
+
+def _inferred_numpy_func_call(func_name, *func_args):
+    node = builder.extract_node(
+        f"""
+    import numpy as np
+    func = np.{func_name:s}
+    func({','.join(func_args):s})
+    """
+    )
+    return node.infer()
+
+
+@pytest.mark.skipif(not HAS_NUMPY, reason="This test requires the numpy library.")
+@pytest.mark.parametrize(
+    "method, args",
+    [
+        ("einsum", "ii, np.arange(25).reshape(5, 5)"),
+    ],
+)
+def test_numpy_function_calls_inferred_as_ndarray(method, args):
```

Comment:
```suggestion
def test_numpy_function_calls_inferred_as_ndarray(method: str, args: str) -> None:
```

Reply:
This isn't relevant if we remove the parametrize decorator.